core/templates/components/profile-link-directives/profile-link-image.component.ts

```diff
@@ -29,9 +29,12 @@ import { ProfileLinkImageBackendApiService } from
   styleUrls: []
 })
 export class ProfileLinkImageComponent implements OnInit {
-  @Input() username: string;
-  profileImageUrl: string;
-  profilePicture: string;
+  // These properties are initialized using Angular lifecycle hooks 
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1.
+  @Input() username!: string;
+  profileImageUrl: string | undefined;
+  profilePicture: string | undefined;
```

Comment:
Why these can be undfined?

Reply:
since they were not preceded by a decorator, the properties are not, be default, initialized in the lifecycle hooks but in this case, they were. Have corrected.